projects/marketing-site/components/Team/CoreTeam.tsx

```diff
@@ -1,31 +1,30 @@
-import Image from "next/image";
-
-import distractionChadBunny from "@/images/chadbit.jpg";
-import dicaso from "@/images/dicaso.jpg";
-import weston from "@/images/pfer-weston.png";
-import commonLang from "@/lang/common.json";
+import distractionChadBunny from '@/images/chadbit.jpg';
+import dicaso from '@/images/dicaso.jpg';
+import jason from '@/images/jason-factor-avatar.png';
+import commonLang from '@/lang/common.json';
+import Image from 'next/image';
 import { useRouter } from 'next/router';
 
 const people = [
   {
-    name: "DistractionBoy",
-    role: "Engineering Lead",
+    name: 'DistractionBoy',
+    role: 'Engineering Lead',
     imageUrl: distractionChadBunny,
-    twitterUrl: "https://twitter.com/DistractionBoy_",
+    twitterUrl: 'https://twitter.com/DistractionBoy_',
     linkedinUrl: null,
   },
   {
-    name: "Dicaso",
-    role: "Creative Director",
+    name: 'Dicaso',
+    role: 'Creative Director',
     imageUrl: dicaso,
-    twitterUrl: "https://twitter.com/Dicaso5",
+    twitterUrl: 'https://twitter.com/Dicaso5',
     linkedinUrl: null,
   },
   {
-    name: "Weston Nelson",
-    role: "Treasury Management",
-    imageUrl: weston,
-    twitterUrl: "https://twitter.com/westonnelson",
+    name: 'Jason Factor',
+    role: 'anhfactor',
+    imageUrl: jason,
```

Comment:
could we still use https://avatars.githubusercontent.com/u/13186215?v=4 

or is that too small for the space? it'd be cool if we could not have to keep that image file in the repo if we don't need to.

Reply:
I get this error when I run it locally with that.
![image](https://user-images.githubusercontent.com/7431851/212419139-e5bb30b4-aa93-484c-a53a-64fe929d71d9.png)

Reply:
@BambooXLotus  To fix this error, in "./projects/marketing-site/components/Team/CoreTeam.tsx", add additional "width={80}" and "height={80}" properties in <Image> tag like this

`<Image
                    className="mx-auto h-40 w-40 rounded-full xl:h-56 xl:w-56"
                    src={person.imageUrl}
                    alt=""
                    priority
                    width={80}
                    height={80}
                  />`
I have checked and it worked!

Reply:
It works but it seems to be getting a low quality Image. I will look more into this tommorow.

![image](https://user-images.githubusercontent.com/7431851/212442914-f26bdf2a-f17d-4d9b-99c2-7872e2889674.png)

Reply:
let's just leave it as-is. i don't like pixelated images / low quality images